src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -590,6 +590,111 @@ static void insertConvTransposeSpatialDim(SmallVectorImpl<int64_t> &outputDims,
   }
 }
 
+//===----------------------------------------------------------------------===//
+// Support function to emit a warning if an operator attibute value is greater
+// than a preset limit.
+//===----------------------------------------------------------------------===//
+
+static InFlightDiagnostic verifyAttributeLimit(
```

Comment:
We should be specific here about what kind of attribute. Otherwise, it looks like the limit is for all attributes, which is not true, for example, `value_int` attribute in constant can be any integer constant. Perhaps, change the function name to a more specific one, e.g. verifyAxisAttributeLitmit.